db/migrate/20160203195736_remove_docker_image_schema.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveDockerImageSchema < ActiveRecord::Migration
```

Comment:
To add to the conversation, based on how we've approached prior data changes during upgrades, if changes need to be made in a backend system try to put that into a rake task so that migrations stay active record focused. Upgrade rake tasks can be run at any point during upgrade. Today, we have both pre (https://github.com/Katello/katello-installer/blob/master/hooks/pre/30-upgrade.rb) and post (https://github.com/Katello/katello-installer/blob/master/hooks/post/30-upgrade.rb) upgrade steps (pre- post- being when they are run relative to the installer itself).